From patchwork Fri Apr 17 00:12:29 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Fabio Estevam <festevam@gmail.com>
X-Patchwork-Id: 1271950
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=2a01:238:438b:c500:173d:9f52:ddab:ee01; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=Y8DIhmdn; dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de
 [IPv6:2a01:238:438b:c500:173d:9f52:ddab:ee01])
 (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
 key-exchange X25519 server-signature RSA-PSS (4096 bits))
 (No client certificate requested)
 by ozlabs.org (Postfix) with ESMTPS id 493Gkp6pfmz9sRN
 for <incoming@patchwork.ozlabs.org>; Fri, 17 Apr 2020 10:12:50 +1000 (AEST)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
 by phobos.denx.de (Postfix) with ESMTP id 4393381A29;
 Fri, 17 Apr 2020 02:12:43 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="Y8DIhmdn";
 dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id 2EC4B81BED; Fri, 17 Apr 2020 02:12:19 +0200 (CEST)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE autolearn=ham
 autolearn_force=no version=3.4.2
Received: from mail-qk1-x732.google.com (mail-qk1-x732.google.com
 [IPv6:2607:f8b0:4864:20::732])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id C9BC780652
 for <u-boot@lists.denx.de>; Fri, 17 Apr 2020 02:12:09 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=festevam@gmail.com
Received: by mail-qk1-x732.google.com with SMTP id g74so669786qke.13
 for <u-boot@lists.denx.de>; Thu, 16 Apr 2020 17:12:09 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025; 
 h=from:to:cc:subject:date:message-id:in-reply-to:references;
 bh=hgD3mVzreEvFgDxQ+B8IpBJTz5pT5ryD9lON1T6sUpo=;
 b=Y8DIhmdnHZMdHNKwieMtQtDQK6FVRExtQi5iROfJoCFrYDzFl2jgbXDkIN2h8fxZPW
 I53GJrZGkeSENp49nkS0SJE3FcvV7qUfi3I32Ye0mfrMdt7aYmpV9ITCb+6GFn8bR1y/
 MU1eP7LR6qMAmWn6F9IpHiUkKzlVxWn/QyuL4ST+79fYy8pkXf29sBkK5UOg0DYEvd3v
 0ikIRFfqQls+qneFJmMzBjAE5h85NkFUJlcMHXaB2nCS0uRXpOcRYNO/BVntvM3xsS/R
 qOul2C/hMkE4Vdc2J09CiO5a0HFr2dfrZofjhgIqpZkE+yofMXfH9I3U3q7NVaKACnpu
 Q2Hg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references;
 bh=hgD3mVzreEvFgDxQ+B8IpBJTz5pT5ryD9lON1T6sUpo=;
 b=DJ1b+vWutdCtEsMdKJ4gnBlUlKH2EmqOvhtZmb/JUy77DWsVdbeZGAbjqaPLg6GGec
 i2hQ+OqguIr7Cr0/xg/2xKDATvqPXaBqVZbB5WwQ6+KrxF5AVhwfMeRXvXEyymOwDD/o
 zRsjRPnQIfWRstkb0x/zrOVdetyYhbRFQNcba6SPkULPLSSYdBlYp2Ds9NL9q2wqJkmr
 4dHKt+Syb/I3U+bMMTsivw3LnFR4lOzGyYBrxM49WGNwFsTVMxyyJ9ovcMsVG/Zgoalh
 z1m7cKP5W6Rym4+uMQOCPAkkC8qGJFPYZ9LCbQBHNSdchERdmPDGkxp3aKENQTCT9/qE
 C5/Q==
X-Gm-Message-State: AGi0Puao9rA0NJzdC9xV862VHjxDueICjXp60DK5eu/DLBaKcd/lm55E
 GtOGiWmwM4kDRtCkkgUCHr8=
X-Google-Smtp-Source: APiQypJ4d5Yr/+Yd0yMMFCQmvnoim/jK416TWs5Zv9kQZEPOkMvfFjMf3974Ge1H96+y7DJp5/oZdg==
X-Received: by 2002:ae9:ed04:: with SMTP id c4mr679049qkg.367.1587082328598;
 Thu, 16 Apr 2020 17:12:08 -0700 (PDT)
Received: from localhost.localdomain ([2804:14c:482:271:809b:81f0:1d41:9d6b])
 by smtp.gmail.com with ESMTPSA id
 x5sm16987996qtm.81.2020.04.16.17.12.06
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Thu, 16 Apr 2020 17:12:07 -0700 (PDT)
From: Fabio Estevam <festevam@gmail.com>
To: sbabic@denx.de
Cc: hs@denx.de, derek@ihtfp.com, u-boot@lists.denx.de,
 Fabio Estevam <festevam@gmail.com>
Subject: [PATCH 3/5] wandboard: Do not print error when PMIC is not present
Date: Thu, 16 Apr 2020 21:12:29 -0300
Message-Id: <20200417001231.11406-3-festevam@gmail.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20200417001231.11406-1-festevam@gmail.com>
References: <20200417001231.11406-1-festevam@gmail.com>
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.30rc1
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.2 at phobos.denx.de
X-Virus-Status: Clean

On wandboard variants prior to revD1, there is no PMIC populated, so
do not print an error when the reading of the device ID register fails.

Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
 board/wandboard/wandboard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/wandboard/wandboard.c b/board/wandboard/wandboard.c
index 5725c5816c..df454d8f17 100644
--- a/board/wandboard/wandboard.c
+++ b/board/wandboard/wandboard.c
@@ -369,7 +369,7 @@ int power_init_board(void)
 
 	reg = pmic_reg_read(dev, PFUZE100_DEVICEID);
 	if (reg < 0) {
-		printf("pmic_reg_read() ret %d\n", reg);
+		debug("pmic_reg_read() ret %d\n", reg);
 		return 0;
 	}
 	printf("PMIC:  PFUZE100 ID=0x%02x\n", reg);
